modules/programs/watson.nix

```
@@ -0,0 +1,96 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.watson;
+
+  iniFormat = pkgs.formats.ini { };
+
+  configDir = if pkgs.stdenv.isDarwin then
+    "Library/Application Support"
+  else
+    config.xdg.configHome;
+
+in {
+  options = {
+    programs.watson = {
+      enable = mkEnableOption "watson, a wonderful CLI to track your time!";
```

```suggestion
      enable = mkEnableOption "watson, a wonderful CLI to track your time";
```
A period is automatically appended.

---

thanks!